integration/nwo/common/context/context.go

```diff
@@ -39,7 +39,8 @@ type Context struct {
 	hostByOrdererID    map[string]string
 	extensionsByPeerID map[string]api.Extensions
 
-	ViewClients             map[string]api.ViewClient
+	ViewClients             map[string]api.GRPCClient
+	WebClients              map[string]api.WebClient
```

Comment:
mh ... I think these terms are not really good here. 

What is a ViewClient? Something that can trigger view actions from remote, right? We have two implementations for that a) GRPC and b) web. What if we add another impl in the future?

Is there a need that both impls are used at the same time?  

We are giving up a bit of abstracting here. For instance, `ViewIdentities` may also have different implementations ... we could have idemix, or whatever, right?

Reply:
So could we also try to hide the impl of the actual view client here and let the system figure out via configuration, if we want to start the FSC node with GRPC or web support, or both?